views/js/previewer/component/qtiItem.js

```diff
@@ -59,7 +59,7 @@ define([
                 communicator: {
                     id: 'request',
                     module: 'core/communicator/request',
-                    bundle: 'loader/vendor.min',
+                    bundle: Object.keys(config.previewers).length === 1 ? 'loader/vendor.min' : null,
```

Comment:
```suggestion
                    bundle: Object.keys(config.previewers || {}).length === 1 ? 'loader/vendor.min' : null,
```
fallback, just in case config will be missed for some reason

Reply:
done